app/forms/webauthn_setup_form.rb

```diff
@@ -52,6 +52,15 @@ def consume_parameters(params)
     @client_data_json = params[:client_data_json]
     @name = params[:name]
     @platform_authenticator = (params[:platform_authenticator].to_s == 'true')
+    @transports_by_validity = partition_transports_by_validity(params[:transports]&.split(','))
```

Comment:
one more idea here:
```suggestion
    @transports, @invalid_transports = partition_transports_by_validity(params[:transports]&.split(','))
```
that way we don't have to qualify each access with `transports_by_validity[:valid]` etc throughout the file?

Reply:
and we'd have to update the method to return an array tuple instead of hash, but same same

Reply:
Went with something like this in 813b0582c. Maybe got a little carried away with the `.method` usage vs. a simple block 🤷

Reply:
yeah I'd have just done the block form? 🤷

Reply:
Yeah, let's just do that. c1bd7432d